lib/src/chip/chip-row/chip-row.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, HostBinding, OnInit, Input } from '@angular/core';
+import { ChipComponent } from '../chip.component';
+import { FilterButtonComponent } from '../filter-button/filter-button.component';
+
+@Component({
+    selector: 'hc-chip-row',
+    templateUrl: './chip-row.component.html',
+    styleUrls: ['../chip.component.scss']
+})
+export class ChipRowComponent implements OnInit {
+    @HostBinding('class') hostClass = 'chip-row';
+    @Input() singleRow: boolean = false;
```

Comment:
just being picky about the name but maybe change to wrap.

You'll also want to create a getter and setter for the input because you want the user to be able to assign a string to this input and you'll need to convert that to a boolean. You can check out button.component.ts's disabled input